npm/private/npm_translate_lock_generate.bzl

```diff
@@ -217,7 +236,9 @@ def npm_link_targets(name = "node_modules", package = None):
     npm_link_all_packages_bzl = [
         """\
 # buildifier: disable=function-docstring
-def npm_link_all_packages(name = "node_modules", imported_links = []):
+def npm_link_all_packages(name = "node_modules", imported_links = [], prod = False, dev = False):
```

Comment:
I think we should drop this API and only include the `npm_link_targets` one to start